www_src/components/snackbar/snackbar.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react/addons');
+
+var Snackbar = React.createClass({
+  getInitialState: function() {
+    return {
+      message: this.props.message,
+      hidden: true
+    };
+  },
+  render: function () {
+    var className = "snackbar";
+    if (this.state.hidden) {
+      className = "hidden " + className;
+    }
+    return <div className={className}>{ this.state.message }</div>;
```

Comment:
hidden doesn't animate nicely, while `height` does. The design docs dictate this has to slide out after 3 seconds.